docs/plugins/http.md

```diff
@@ -242,6 +243,20 @@ Following are examples that illustrate the syntax usage for the HTTP task.
    - log: "Response received: ${jsonResponse.content}"
 ```
 
+##### Runtime v2 Syntax for GET Request
```

Comment:
I think this can be made more broad. The "runtime v2 changes" aren't so much about the http task's parameters directly changing, but all plugins in runtime v2 return a result in the same way now. e.g. `out` didn't move, there were two places to use `out` in runtime-v1: 1) the http task's `out` parameter controlled what variable the task put its result in and 2) the `out` parameter for _any_ task or flow call. We just removed the first one.
```yaml
# double-out fun
- task: http
  in:
     # ...
     out: response  # this was a bad idea to begin with
   out:
     renamedOutVar: ${response}
- log: "got: ${renamedOutVar}"
```

I think a slightly more robust explanation would get the _actual_ changes clarified. I'm not married to the verbiage here or formatting per se, but this seems more clear to me:


## Runtime V2 Changes

### Removed Input Parameters

- `out` - no longer required to specify an out-variable name

### Removed Output Values

- `success` - Use standardized `ok` value
- `errorString` - Use standardized `error` value

### Standardized `out` result.

All runtime-v2 tasks return their data in the same fashion via the `task` call's `out` attribute

```yaml
- task: myTask
  in:
    aParam: a-value
  out: resultVarName
```

All data returned by runtime-v2 tasks contain the same basic attributes `ok` and `error` along with per-task values.

```json
{
  "ok": true,
  "error": "error message when 'ok' is false",
  "customValue": "the value"
}
```

This remove removes the need for the HTTP task's `out` parameter in favor of relying solely on the generic task call's `out` attribute. Likewise, `success` and `errorString` are replaced by the standardized `ok` and `error` output values. The rest of the HTTP task's out-values contained within the returned `out` variable remain the same.

```yaml
- task: http
  in:
   method: GET
    url: "https://api.example.com:port/path/endpoint"
    response: json
  out: response
- if: ${response.ok}
  then:
    - log: "Response payload: ${resource.prettyPrintJson(response.content)}"
  else:
   - log: "HTTP call failed with code ${response.statusCode}: ${response.error}"
   - exit
```

Reply:
Well I was doing this more from the perspective of users who get stuck on the http task like our friend. what you are suggesting is more generic and it should probably go in here - https://concord.walmartlabs.com/docs/getting-started/tasks-v2.md OR here https://concord.walmartlabs.com/docs/processes-v2/tasks.html. 

But again, adding it in the v2 docs won't solve the problem because v2 users are still inclined to use the same v1 task docs and not go to the "under development" section ever. Maybe we can keep both? I can update this PR with your text in the v2 section if you agree

Reply:
I think that's a fair compromise. Add the general explanation to the migration page (not the tasks page, the syntax applies to more than just `task` calls). The first thing the v2 info on the http task's docs should do is link to that though. I get that an example is helpful, but actually understanding the syntax helps in more cases than just one plugin.

At some point the docs should just be in v2 syntax though 🤔